packages/common-ui-web/test/components/InspectionGallery/InspectionGalleryItemCard/InspectionGalleryItemCard.test.tsx

```diff
@@ -164,7 +164,7 @@ describe('InspectionGalleryItemCard component', () => {
     const { unmount } = render(<InspectionGalleryItemCard {...props} />);
 
     expect(useInspectionGalleryItemStatusIconName).toHaveBeenCalled();
-    expectPropsOnChildMock(Icon, { icon: useInspectionGalleryItemStatusIconName({} as any) });
+    expectPropsOnChildMock(Icon, { icon: useInspectionGalleryItemStatusIconName({} as any)! });
```

Comment:
Even in tests, I really don't like using `!`, try to find a better solution

Reply:
Instead of this, we need to specify the exact type using the 'as' keyword to inform TypeScript about a different type But that also have it downsights.